lib/browser/api/dialog.js

```diff
@@ -159,10 +168,27 @@ module.exports = {
       filters = []
     }
 
+    if (message == null) {
+      message = ''
+    } else if (typeof message !== 'string') {
+      throw new TypeError('Message must be a string')
+    }
+
+    if (nameFieldLabel == null) {
+      nameFieldLabel = ''
+    } else if (typeof nameFieldLabel !== 'string') {
+      throw new TypeError('Name field label must be a string')
+    }
+
+    if (showsTagField == null) {
+      showsTagField = false
```

Comment:
Looks like the default OS value of this is `true` if unset so I think this should default to `true` to not break backwards-compatibility.

https://developer.apple.com/reference/appkit/nssavepanel/1525589-showstagfield?language=objc